### ISSUE TITLE
weszlo.com

### ISSUE THREAD

**Issue by MaksymilianWars** [reek/anti-adblock-killer#2683]
I'm using Chrome + uBlock Origin, with Anti-Adblock-Killer filter on, still cannot access weszlo.com without turning off the adblocker.

Any hints?

![screen shot 2017-01-12 at 13 36 59](https://cloud.githubusercontent.com/assets/25079637/21890174/bde73ff6-d8cc-11e6-8939-f1a76ce48d07.png)

Many thanks!
M

**Comment by uBlock-user** [comment]
add - `weszlo.com##script:inject(bab-defuser.js)`

**Comment by mapx-** [comment]
add in ubo:
```
||weszlo.com$inline-script
weszlo.com###bsa-html
```

**Comment by MaksymilianWars** [comment]
Thank you! I tried the first method and it works perfectly.

I also tried adding the filter on my iPhone (iOS 10) + Safari, where I'm using Adguard, as an adblocker, but it won't work. Any hints?

Very best,
M

**Comment by uBlock-user** [comment]
That filter is for uBO only, won't work for any other generic adblocker.

**Comment by xxcriticxx** [comment]
@MaksymilianWars zapraszam do nas https://github.com/MajkiIT/polish-ads-filter/issues

**Comment by MaksymilianWars** [comment]
@uBlock-user, fair enough! Thanks for your support so far

Weszlo.com added additional protection now, I'm no longer seeing the BlockAdblock warning, but all the articles are trimmed with "[...]" somewhere in the first paragraph and it won't load the whole page without turning off uBlock. Example page: weszlo.com/2017/01/16/meczow-ktorych-wystawilem-dobra-note-bylo-malo/

uBlock turned off:
![screen shot 2017-01-16 at 16 51 48](https://cloud.githubusercontent.com/assets/25079637/21989757/72f7a7e0-dc0c-11e6-801b-646d4d2c6ca2.png)

uBlock turned on (with custom filters you provided earlier on):
![screen shot 2017-01-16 at 16 52 23](https://cloud.githubusercontent.com/assets/25079637/21989719/56150852-dc0c-11e6-91d4-59c0fa3598aa.png)

Any ideas?

**Comment by mapx-** [comment]
add
```
@@||weszlo.com^$generichide
weszlo.com###bsa-html
```

**Comment by MaksymilianWars** [comment]
@mapx-, thanks! When only using the code you provided, I'm again getting the BlockAdblock overlay. Please see below. Any ideas how to solve this?

![screen shot 2017-01-16 at 22 27 19](https://cloud.githubusercontent.com/assets/25079637/21999576/255dce90-dc3b-11e6-9511-ee657e7490f5.png)

User filters SS:
![screen shot 2017-01-16 at 22 29 30](https://cloud.githubusercontent.com/assets/25079637/21999608/49d46874-dc3b-11e6-8d2f-6db119214780.png)

**Comment by mapx-** [comment]
add also
`weszlo.com#@##banner_ad`

**Comment by deejayscream** [comment]
Thanks,by helping my friends above,you've helped me as well...:)well done!
But...another polish listings site I've got problems with: www.allegro.pl
Searched e.g. addias shoes,got some listings,but each of them can open only in a new tab...:
Checked ebay as well to compare,works just fine.

![zrzut ekranu 2017-01-17 o 02 24 57](https://cloud.githubusercontent.com/assets/25163477/22004497/3c01b000-dc5c-11e6-97b2-bfec95d45535.png)

Here is my filters list + POL + theese from #2683 topic to avoid AB detector
weszlo.com##script:inject(bab-defuser.js)
||weszlo.com$inline-script
weszlo.com###bsa-html
@@||weszlo.com^$generichide
weszlo.com#@##banner_ad

![zrzut ekranu 2017-01-17 o 02 26 45](https://cloud.githubusercontent.com/assets/25163477/22004603/49e65076-dc5d-11e6-9496-f344a2fb7f86.png)

What is causing a problem??
Thanks for any hints.

**Comment by mapx-** [comment]
I don't understand your problem, it's normal opening a new tab

**Comment by deejayscream** [comment]
Yeah,ok,it's normal.But it's not normal,that this is the only way for me to open it :)
I can't click on a link and see the listing,like something is blocking,If I turn off ubo on this site,everything is fine.

**Comment by MaksymilianWars** [comment]
Hi @mapx-, thanks again! 👍  Still no luck unfortunately. Same message as earlier, BlockAdblock overlay. Here's the screenshot of my user filters:
![screen shot 2017-01-17 at 11 47 12](https://cloud.githubusercontent.com/assets/25079637/22017454/cb3849fc-dcaa-11e6-9921-0b860d5b710f.png)

**Comment by mapx-** [comment]
tested again, using only
```
@@||weszlo.com^$generichide
weszlo.com###bsa-html
weszlo.com#@##banner_ad
```
it's ok for me.
If you are using any other blocking extension (ghostery, disconnect, ...), disable it, test again.

**Comment by deejayscream** [comment]
I don't think anti adblock filters for weszlo.com can causing the problems on allegro.pl 
Using only clicktoplugin extension on safari,plus ubo with filters listed above.
Problem is still existing.

**Comment by MaksymilianWars** [comment]
@mapx-, I disabled Ghostery, plus updated all uBlock filters, restarted Chrome. But whenever I enter weszlo.com I'm getting DisableAdblock overlay after a few seconds.

Could it be that I'm not using one of required filter lists?

![screen shot 2017-01-17 at 15 27 35](https://cloud.githubusercontent.com/assets/25079637/22024210/831fc45a-dcc9-11e6-9a9d-d9025cecc15a.png)

![screen shot 2017-01-17 at 15 25 49](https://cloud.githubusercontent.com/assets/25079637/22024168/5c3bd9be-dcc9-11e6-8da2-fc4a4c9a881b.png)

**Comment by mapx-** [comment]
retested with all your lists, it works for me ..

**Comment by MaksymilianWars** [comment]
@mapx-, ok. The issue is probably somewhere deeper. Thanks so much for your help.

I looked around, and just found this solution elsewhere: https://github.com/MajkiIT/polish-ads-filter/issues/1997. Luckily it works for me:  `weszlo.com##.adv:style(height: 1px !important; width: 1px !important;)`

**Comment by MaksymilianWars** [comment]
#https://github.com/MajkiIT/polish-ads-filter/issues/1997
